nise/generators/oci/oci_constants.py

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""OCI Report Constants"""
+import random
+from dataclasses import dataclass
+from dataclasses import field
+
+from faker import Faker
+
+
+fake = Faker()
+
+
+@dataclass(frozen=True)
+class OCIRegions:
+    """OCI Regions"""
+
+    region: str
+    domain: str
+
+
+def random_region_domain():
+    """get random region domain tuple"""
```

Comment:
This returns an `OCIRegion` object, not a tuple.
```suggestion
    """Return random region domain"""
```